### PR TITLE
BD-3463 Connecting a subdomain docs

### PR DESCRIPTION
**Pull request by rachel-feinberg**
### Summary

My PR adds the Connecting a Subdomain docs to Landing Pages

### Related PRs, issues, or features (optional)

[BD-3463](https://jira.braze.com/browse/BD-3463)

### Feature release date (optional)

- N/A

### Contributor checklist

- [ ] I confirm that my PR meets the following:
    - I signed the [Contribution License Agreement (CLA)](https://www.braze.com/docs/cla).
    - My style and voice follows the [Braze Style Guide](https://docs.google.com/document/u/2/d/e/2PACX-1vTluyDFO3ZEV7V6VvhXE4As_hSFwmnFFdU9g6_TrAYTgH1QmbRoEDDdn5GzKAB9vdBbIdyiFdoaJcNk/pub).
    - My content contains correct spelling and grammar.
    - All links are working correctly.
    - If I renamed or moved a file or directory, I set up [URL redirects](https://www.braze.com/docs/contributing/content_management/redirecting_urls) for each file.
    - If I updated or replaced an image, I did not remove the original image file from the repository. (For more information, see [Updating an image](https://www.braze.com/docs/contributing/content_management/images/#updating-an-image)).
    - If my PR is related to a paid SKU, third party, SMS, AI, or privacy, I have received written approval from Braze Legal.

### Submitting for review

If your PR meets the above requirements, select **Ready for review**, then add a reviewer:
- **Non-Braze contributors:** Add `braze-inc/docs-team` as the reviewer.
- **Braze employees:** Have any relevant subject matter experts (like engineers or product managers) review your work first, then add the [tech writer assigned to your specific vertical](https://confluence.braze.com/pages/viewpage.action?pageId=75039896). If you're not sure which tech writer to add, you can add `braze-inc/docs-team` instead. 

Thanks for contributing! We look forward to reading your work.

### CODE REVIEW

File: _docs/_user_guide/engagement_tools/landing_pages/connect_subdomain.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+1. Go to **Settings** > **Landing Page Settings**.
+2. Enter the subdomain you want to connect and select **Submit**.
+3. Copy and paste the **TXT** and **CNAME** records into the relevant settings of your domain provider account.
+4. Return to the Braze dashboard to verify the connection.
+
+![Landing Page Settings page with one TXT and two CNAME records listed with their respective names and values.][1]
+
+{% alert note %}
+The subdomain connection can take up to 48 hours, depending on your domain provider. We’ll update your **Landing Page Settings** page when the process is complete.
```
Comment (lydia-xie) [inline _docs/_user_guide/engagement_tools/landing_pages/connect_subdomain.md:27]:
```suggestion
Depending on your domain provider, the subdomain connection can take up to 48 hours. When the process is complete, we’ll update your **Landing Page Settings** page.
```
minor suggestion

**Review by lydia-xie — APPROVED** [review]
lgtm! left a small suggestion

File: _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md [hunk 2]
```diff
@@ -133,13 +133,13 @@ No, this isn’t currently available. You can use the Custom Code block in the e
 ### What features are on the roadmap for landing pages? {#roadmap}
 
 Because landing pages are currently in beta, additional features are in development. These include:
+
 * Custom subdomain support
```
Comment (rachel-feinberg) [inline _docs/_user_guide/engagement_tools/landing_pages/drag_and_drop.md:137]:
```suggestion
```

**Comment by rachel-feinberg** [comment]
@jridgwaybraze , @davidbielik Is this ready to merge?

**Comment by davidbielik** [comment]
@rachel-feinberg yes please, thanks

**Review by jridgwaybraze — APPROVED** [review]

**Pull request merged**